packages/components/addon/components/hds/copy/button/index.hbs

```diff
@@ -0,0 +1,22 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<Hds::Button
+  class={{this.classNames}}
+  @size={{this.size}}
+  @isFullWidth={{@isFullWidth}}
+  @text={{@text}}
+  @icon={{this.icon}}
+  @isIconOnly={{@isIconOnly}}
+  @color="secondary"
+  @iconPosition="trailing"
+  {{clipboard
+    text=this.textToCopy
+    target=this.targetToCopy
+    container=this.container
```

Comment:
Good to have this covered here, now that we know it's useful when the Copy components are used in Modals/Flyouts